datalad/support/path.py

```diff
@@ -74,18 +74,17 @@ def robust_abspath(p):
     """
     try:
         return abspath(p)
-    except OSError as exc:
+    except OSError:
         if not isabs(p):
             try:
                 os.getcwd()
-                # if no exception raised it was not the reason, raise original
-                raise
-            except:
+            except Exception:
```

Comment:
ha -- so even logic I added in `0.10.3~18^2` (53cfc2d8ef7d8d268031ef5aee1ae925bd89fd8f) was wrong since I was re-raising and then right away going into `except` for it... odd.  This fixes that now I think. BUT I would keep it 
```suggestion
            except:
```
or  at least
```suggestion
            except BaseException:
```
if possible to retain original effect from `getcwd` crashing one way or another.

Reply:
Bare `except:`/`except BaseException:` will also catch `KeyboardInterrupt` and other things that normally shouldn't be caught.